includes/class-sensei-quiz.php

```diff
@@ -61,8 +61,13 @@ public function __construct( $file = __FILE__ ) {
 	 * @return bool
 	 */
 	public function is_block_based_editor_enabled() {
-		// If custom question types have been registered, disable the block based quiz editor for now.
-		$is_block_based_editor_enabled = ! has_filter( 'sensei_question_types' );
+		if ( ! function_exists( 'is_plugin_active' ) ) {
+			include_once ABSPATH . 'wp-admin/includes/plugin.php';
+		}
+
+		// If custom question types have been registered, or the Classic Editor plugin is activated,
+		// disable the block based quiz editor for now.
+		$is_block_based_editor_enabled = ! has_filter( 'sensei_question_types' ) && ! is_plugin_active( 'classic-editor/classic-editor.php' );
```

Comment:
I wonder if we can use `use_block_editor_for_post_type( 'lesson' )` instead?

Reply:
Tried it but seems to be some sort of load issue as I got a fatal that the function was undefined. Probably not going to diddle around with it more.

Reply:
Ahh it looks like it is just in `wp-admin/includes/post.php`. I get a bit nervous about this kind of plugin slug checks because they're pretty brittle (installed in `classic-editor-2/classic-editor.php`) and it doesn't catch uses of Gutenberg Ramp and alternative plugins. If no other way to sniff, then I guess maybe.

Reply:
We'll likely catch most and those that we don't still have the option to open a ticket or forum post.